src/main/java/org/damap/base/rest/openaire/mapper/OpenAireMapper.java

```diff
@@ -125,6 +127,7 @@ public DatasetDO mapAtoB(String propertyName, QualifierType qualifierType, Datas
    * @return a {@link org.damap.base.rest.dmp.domain.DatasetDO} object
    */
   public DatasetDO mapAtoB(String propertyName, String string, DatasetDO datasetDO) {
+    // TODO: check if file format is provided
```

Comment:
File format is provided as `format`. See target/generated-sources/jaxb/eu/openaire/oaf/Result.java or https://www.openaire.eu/schema/1.0/doc/oaf-result-1_0_xsd.html#result_format. I tested it and the return value was a mime type - although it is not stated that this is always the case.

Reply:
Thank you. I could not find a dataset, where this is set. So I just used the same logic of trying to parse a mime type.

Reply:
Yes i think this reasonable